modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -15,9 +15,9 @@ When you create a `VolumeSnapshot` object, {product-title} creates a volume snap
 * A storage class to provision the storage back end.
 * No pods are using the persistent volume claim (PVC) that you want to take a snapshot of.
 +
-[NOTE]
+[IMPORTANT]
 ====
-Do not create a volume snapshot of a PVC if a pod is using it. Doing so might cause data corruption because the PVC is not quiesced (paused). Be sure to first tear down a running pod to ensure consistent snapshots.
+Creating a volume snapshot of a PVC that is in use by a pod can cause unwritten (cached) data to be excluded from the snapshot. To ensure that all data is written to the disk, delete the pod that is using the PVC before creating the snapshot.
```

Comment:
```suggestion
Creating a volume snapshot of a PVC that is in use by a pod can cause unwritten cached data to be excluded from the snapshot. To ensure that all data is written to the disk, delete the pod that is using the PVC before creating the snapshot.
```
I know IBM Style recommends avoiding parens and I don't think you need one here, but it's your call. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses

Reply:
@brandisher 
Is it technically ok to have "unwritten cached" data?

Reply:
We can say `unwritten or cached` which is technically correct and drops the parens.

Reply:
Based on the discussion with @brandisher , updated it to "unwritten data and cached data" as both are different.